src/test-setup.js

```diff
@@ -36,4 +37,12 @@ global.OC = {
 		return 'en_GB'
 	},
 }
+global.OCA = {
+	Talk: {
+	},
+}
 global.t = jest.fn().mockImplementation((app, text) => text)
+
+Vue.prototype.t = global.t
```

Comment:
Also add n already

Reply:
I wasn't sure how to mock it in a good way without replicating the full logic. I'll just make it return the first text then.